packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -251,6 +252,10 @@ export class EpochTracker implements IPersistedFileCache {
             if (this.fluidEpoch !== undefined) {
                 headers["x-fluid-epoch"] = this.fluidEpoch;
             }
+            const isClpCompliantApp = getOdspResolvedUrl(this.fileEntry.resolvedUrl).isClpCompliantApp;
+            if (isClpCompliantApp) {
+                headers[ClpCompliantAppHeader.isClpCompliantApp] = isClpCompliantApp.toString();
```

Comment:
You need to add in else condition also like we added this:
addHeader("X-RequestStats", clientCorrelationId);

Reply:
Added to else condition in commit 9c39d14